rt/mlc/TBRCTX.MLC

```diff
@@ -116,8 +115,8 @@ LblBRCTG DC    CL5'BRCTG'
 QW       DS    XL16,Xl1            Quadword work and pad
 DW       DS    D,Xl1               Doubleword work and pad
 *
-WTO1     WTO   'After xxxxx loop; R2=xxxxxxxxxxxxxxxx  R3=xxxxxxxxxxxxxx
-               xxx',MF=L
+WTO1     WTO   ' xxxxx loop; R2=xxxxxxxxxxxxxxxx  R3=xxxxxxxxxxxxxxxxx'X
+               ,MF=L
 W1Lbl    EQU   WTO1+4+6,5,C'C'
 W1R2     EQU   WTO1+4+21,16,C'C'
 W1R3     EQU   WTO1+4+42,16,C'C'
```

Comment:
Removing "After" from the WTO shifts everything else in the WTO text left 5 positions. The offsets for the 3 equated values must also be decremented by 5. Replace those 3 lines with
```
W1Lbl    EQU   WTO1+4+1,5,C'C'
W1R2     EQU   WTO1+4+16,16,C'C'
W1R3     EQU   WTO1+4+37,16,C'C'
```

Reply:
Addendum. There is a 17th "x" following "R3=". Also replace the WTO with
```
WTO1     WTO   ' xxxxx loop; R2=xxxxxxxxxxxxxxxx  R3=xxxxxxxxxxxxxxxx',X
               MF=L
```